src/Fulma.Elmish/DatePicker/View.fs

```diff
@@ -97,11 +100,23 @@ module View =
                 Date.Format.localFormat config.Local config.Local.Date.DefaultFormat date
             | None -> ""
         div [ ]
-            [ yield Input.text [ Input.Props [ Value dateTxt
-                                               OnFocus (fun _ -> onFocus config state currentDate dispatch)
-                                               OnClick (fun _ -> onFocus config state currentDate dispatch)
-                                               // TODO: Implement something to trigger onChange only if the value actually change
-                                               OnBlur (fun _ -> let newState = { state with InputFocused = false }
-                                                                onChange config newState currentDate dispatch) ] ]
+            [
+              yield Field.body [] [
```

Comment:
Do we really need the `Field.body` ?

Perhaps we can expose `Field.div ...` directly.

I would remove the both `div` & `Field.body` is the display is still correct.

Reply:
We need the `div` for the calender below, otherwise we would return `ReactElement list` instead of `ReactElement` and without the `Field.body` I get strange effects when the calender is visible. 

![2018-06-05_16-36-26](https://user-images.githubusercontent.com/6583600/40982927-c2c66558-68de-11e8-9d7c-1a3c1c2268fe.gif)

Reply:
Ah yes, I was looking only at the diff and forget about the calendar itself. Oups :)

Ok looks, good I will review it and test it on my side too before publishing thanks for your work :)